examples/tests/voronoi.cpp

```diff
@@ -90,6 +90,40 @@ struct minFunctor {
   }
 };
 
+template <typename T>
+auto shift_range(T& range, int length, int amount) {
+  auto cycled = ranges::view::cycle(range);
+  return ranges::view::slice(cycled, length + amount, 2 * length + amount);
+}
+
+template <typename T>
+auto shift_and_zip(T& range, int length, int k, int m, int n) {
+  // Use transform with identity to force a load and strip the global address space
+  auto i0 = range | ranges::view::transform([](auto a) { return a; });
```

Comment:
This looks like a workaround to the device compiler, have you reported the underlying bug?

Reply:
It is. We discussed this last week and I created issue #16 about this.

Reply:
Sorry, didn't match the workaround with the bug in my mind!